tests/system/feature_store/test_spark_engine.py

```diff
@@ -47,8 +49,9 @@ def custom_setup(self):
         from mlrun.run import new_function
         from mlrun.runtimes import RemoteSparkRuntime
 
+        self._init_env_from_file()
```

Comment:
It's a pitfall. The comment says
```
# Set to True if you want to avoid the image building phase
```
but if you do that, init won't run. So to allow for quick running as instructed, I moved this line out.